Gemfile.lock

```diff
@@ -1,6 +1,3 @@
-GEM
-  specs:
-
```

Comment:
Not sure why this disappeared, maybe because we're not using the block syntax for the `source` anymore? I'm glad it did, though, because it didn't make much sense empty like that.